WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -136,24 +161,31 @@ private extension TotalsView {
         }
     }
 
-    @ViewBuilder func priceFieldView(title: String, formattedPrice: String) -> some View {
-        VStack(alignment: .leading, spacing: .zero) {
-            Text(title)
-            Text(formattedPrice)
-                .font(.title2)
-                .fontWeight(.medium)
+    @ViewBuilder func priceFieldView(title: String, formattedPrice: String?, shimmeringActive: Bool) -> some View {
+        VStack(alignment: .center, spacing: .zero) {
+            Text(title.uppercased())
+                .font(Font.system(size: 16))
+                .fontWeight(.semibold)
+                .foregroundColor(Color.totalsTitleColor)
+            Text(formattedPrice ?? "-----")
+                .font(Font.system(size: 28))
+                .redacted(reason: formattedPrice == nil ? [.placeholder] : [])
+                                .shimmering(active: shimmeringActive)
```

Comment:
This piece is a bit hard to understand, maybe we should move it to a computed variable and itemize more clearly when would be .redacted and/or .shimmering

Reply:
It is redacted if there is no price yet and shimmering if it is loading. How would you make it clearer?

Reply:
Good question. I'm thinking if just by extracting it outside of the View into a computed variables would make it more readable, and we could reuse it, something like:
```
View()
  .redacted(reason: redactedActive)
```

I'm not quite sure how would I simplify the `reason: formattedPrice == nil ? [.placeholder] : []` bit, I've attempted a few approaches but it is what it is 😅